tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1772,6 +1772,14 @@ def testSqrt(self):
 
 class RoundingTest(tf.test.TestCase):
 
+  def _compare_values(self, x, y=None, use_gpu=True):
```

Comment:
You can take out the use_gpu arguments.  If you pass nothing as use_gpu, it will use GPU if available and CPU otherwise.